module/Database/src/Validator/EuroIban.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Database\Validator;
+
+use Laminas\Validator\Iban;
+
+/**
+ * Extension of the IBAN validator that forces that a country pays in euros
+ */
+class EuroIban extends Iban
+{
+    protected $allowNonSepa = false;
+
+    /**
+     * The SEPA country codes that have the EURO
+     *
+     * @var string[] ISO 3166-1 codes
+     */
+    protected static $sepaCountries = [
+        'AT',
+        'AD',
+        'BE',
+        'CY',
+        'EE',
+        'FI',
+        'FR',
+        'DE',
+        'GR',
+        'IE',
+        'IT',
+        'LV',
+        'LT',
+        'LU',
+        'MT',
+        'MC',
+        'NL',
+        'PT',
+        'SK',
+        'SI',
+        'ES',
+        'SM',
```

Comment:
I assume the list is sorted based on the English name of each country, but that would mean that `AT` and `AD` should be switched, `SM` should go after `PT`, and `VA` (Vatican City) is missing.